src/renderer/modals/FullNode/steps/StepSatStack.js

```diff
@@ -0,0 +1,189 @@
+// @flow
+import React, { useCallback } from "react";
+import { useObservable } from "@ledgerhq/live-common/lib/observable";
+import { statusObservable } from "@ledgerhq/live-common/lib/families/bitcoin/satstack";
+import Box from "~/renderer/components/Box";
+import Text from "~/renderer/components/Text";
+import BigSpinner from "~/renderer/components/BigSpinner";
+import InfoBox from "~/renderer/components/InfoBox";
+import { Trans } from "react-i18next";
+import Button from "~/renderer/components/Button";
+import IconCheck from "~/renderer/icons/Check";
+import IconCross from "~/renderer/icons/Cross";
+import { CheckWrapper, CrossWrapper } from "~/renderer/modals/FullNode";
+import { openURL } from "~/renderer/linking";
+import { urls } from "~/config/urls";
+
+const SatStack = ({
+  satStackDownloaded,
+  setSatStackDownloaded,
+}: {
+  satStackDownloaded: boolean,
+  setSatStackDownloaded: boolean => void,
+}) => {
+  const latestStatus = useObservable(statusObservable, { type: "satstack-disconnected" });
```

Comment:
mmh I don't think the initial state should be `{ type: "satstack-disconnected" }`
it should be actually null/undefined and in that case you would render a spinner because we don't actually know yet the status. (see #sat-stack discussion)